source/ada/lsp-ada_handlers.adb

```diff
@@ -247,6 +247,13 @@ package body LSP.Ada_Handlers is
       URI  : LSP.Types.LSP_String) return LSP.Types.LSP_String;
    --  Turn URI into path
 
+   function Get_Project
```

Comment:
LSP.Ada_Documents seems a better package to put that function (might be reused somewhere else in the future)